examples/connect-app-teleop-react/src/env.ts

```diff
@@ -0,0 +1,41 @@
+export interface BuildEnvironment {
+  baseUri: string;
+  orgId: string;
+  isDev: boolean;
+  auth:
+  | { case: 'api_key'; apiKeyId: string; apiKeySecret: string }
+  | { case: 'third_party'; clientId: string };
+}
+
+let env: BuildEnvironment | undefined;
+
+// import.meta.env is validated at build time in vite.config.ts
+export const getEnv = (): BuildEnvironment => {
+  if (env) {
+    return env;
+  }
+  const buildEnv = {
+    baseUri: import.meta.env.VITE_BASE_URI === '' ? 'http://localhost:9000' : import.meta.env.VITE_BASE_URI,
+    orgId: import.meta.env.VITE_APP_ORG_ID,
+    isDev: import.meta.env.DEV,
+  } as BuildEnvironment;
```

Comment:
Do we need to cast or can we just declare the type?
```suggestion
  const buildEnv: BuildEnvironment = {
    baseUri: import.meta.env.VITE_BASE_URI === '' ? 'http://localhost:9000' : import.meta.env.VITE_BASE_URI,
    orgId: import.meta.env.VITE_APP_ORG_ID,
    isDev: import.meta.env.DEV,
  };
```